app/views/provider/debt-management/990175001/debt-details.html

```diff
@@ -72,66 +70,45 @@ <h2 class="nhsuk-card__heading nhsuk-u-margin-bottom-0">
           <div class="nhsuk-form-group">
             <fieldset class="nhsuk-fieldset">
               <legend class="nhsuk-fieldset__legend nhsuk-u-font-weight-bold">
-                View by
+                Choose the timeframe to view payments and deductions.
               </legend>
               <div class="nhsuk-radios">
                 <div class="nhsuk-radios__item">
-                  <input class="nhsuk-radios__input" id="debt-monthly-yes" name="debt-period-select"
-                      type="radio" value="monthly" checked="checked">
-                  <label class="nhsuk-label nhsuk-radios__label" for="debt-monthly-yes">
-                    Monthly view
+                  <input class="nhsuk-radios__input" id="debt-last-month" name="debt-period-select"
+                    type="radio" value="monthly" checked="checked">
+                  <label class="nhsuk-label nhsuk-radios__label" for="debt-last-month">
+                    Last month
+                  </label>
+                </div>
+                <div class="nhsuk-radios__item">
+                  <input class="nhsuk-radios__input" id="debt-3-month" name="debt-period-select"
+                    type="radio" value="yearly">
+                  <label class="nhsuk-label nhsuk-radios__label" for="debt-3-month">
+                    Last 3 month
                   </label>
                 </div>
                 <div class="nhsuk-radios__item">
-                  <input class="nhsuk-radios__input" id="debt-monthly-no" name="debt-period-select"
-                      type="radio" value="yearly">
-                  <label class="nhsuk-label nhsuk-radios__label" for="debt-monthly-no">
-                      Date range
+                  <input class="nhsuk-radios__input" id="debt-6-month" name="debt-period-select" type="radio" value="yearly">
+                  <label class="nhsuk-label nhsuk-radios__label" for="debt-6-month">
+                    Last 6 month
                   </label>
                 </div>
               </div>
             </fieldset>
+            <div class="nhsuk-u-margin-top-5">
```

Comment:
Taking consideration of button (as inline element alignment gets pulled/distort if sibling inline elements etc. ). Now here it will not matter as sibling block elements(I.e fieldset ). But for consistency have attached this margin on div as other places.   
<div class="nhsuk-u-margin-top-5"> or <button class="nhsuk-u-margin-top-5"> will have the same effect.

Reply:
I don't get your point here. Adding a margin-top shouldn't effect surrounding elements. Nesting the button in a div.. on its own for the purpose of adding 1 class to the button isn't necessary and a bit messy. The whole purpose of utility classes like "nhsuk-u-margin-top-5" is so we can have some customization on components either changing the spacing, font style etc.

Reply:
let's discuss it.